README.md

```diff
@@ -25,6 +25,20 @@ React.render(
 Make sure to include `json-inspector.css` in your stylesheet. Feel free to
 override or amend default styles, for instance, when using a dark background.
 
+### Run the example
+
+```bash
+npm install -g browserify watchify
```

Comment:
Not a problem, but there's no need to install those packages globally, since [`PATH` is updated](https://docs.npmjs.com/misc/scripts#path) while running npm scripts.